pkg/proc/native/proc.go

```diff
@@ -326,7 +326,7 @@ func openRedirects(redirects [3]string, foreground bool) (stdin, stdout, stderr
 			return dflt
 		}
 		var f *os.File
-		f, err = os.Create(path)
+		f, err = os.OpenFile(path, os.O_WRONLY|os.O_CREATE, 0o666)
```

Comment:
This is a backwards incompatible change because you are not passsing O_TRUNC.

Reply:
Will add it - I still need to check whether `O_TRUNC` has any bad effect when opening a fifo.